src/LaTeX.jl

```diff
@@ -182,12 +195,17 @@ function processitem(p, item::TOC, indent)
     ["\\tableofcontents"]
 end
 
-function processitem(p, item::Abstract, indent)
-    r = Any["\\begin{abstract}"]
-    append!(r, processitem(p, item.content, indent))
-    push!(r, "\\end{abstract}")
+function processitem(p, item::Block, indent; options = [])
```

Comment:
Are these `options` passed in from anywhere? It seems a bit inconsistent to me that only this method supports options.